uaclient/cli.py

```diff
@@ -1553,7 +1553,8 @@ def action_status(args, *, cfg):
     show_beta = args.all if args else False
     token = args.simulate_with_token if args else None
     active_value = ua_status.UserFacingConfigStatus.ACTIVE.value
-
+    if is_contract_changed(cfg):
```

Comment:
Maybe this call here should be performed inside a `try..except` block. My rationale is that if this function fails for some reason, we should still show the status output if we can.

What do you think @renanrodrigo @orndorffgrant ?

Reply:
Makes sense.

Reply:
Should we just print to the log file or also show it to the client?

Reply:
Just log it imho - If we could not detect any contract change for any reason (network? etc) then we just show the status. I'd be worried to inform the user that something happens only in a command like `refresh`, where there is expectation of changes to be applied